templates/Procfile

```diff
@@ -1,2 +1,2 @@
-web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
+web: bundle exec rails server Puma -p ${PORT:-3000} -c ./config/puma.rb
```

Comment:
I might not have explained myself correctly. Using Puma directly runs the server unaware of the rails environment: this modifies logging behaviour, if I'm not mistaken.